src/main/java/com/fewlaps/quitnowcache/QNCache.java

```diff
@@ -195,4 +198,29 @@ private String getEffectiveKey(String key) {
         }
         return key;
     }
+    
+    /**
+     * Find 
+     */
+    List<String> findStartsWith(String someCuquiWord) {
+    	final String key = getEffectiveKey(someCuquiWord);
+
+    	return Collections.list(cache.keys())
+    			.stream()
+    			.filter(s -> s.startsWith(key))
+    			.collect(Collectors.toList());
+    }
+    
+    /**
+     * Finding coincidences
+     */
+    List<String> findStartsWithIfAlive(String someCuquiWord) {
+    	final String key = getEffectiveKey(someCuquiWord);
+    	final long now = now();
+    	
+    	return Collections.list(cache.keys())
+    			.stream()
+    			.filter(s -> s.startsWith(key) && cache.get(key).isAlive(now))
```

Comment:
No prob! I also forgotten it :nerd_face: but I noticed it when Travis failed building the project :·)